src/BenchmarkDotNet/ConsoleArguments/ConfigParser.cs

```diff
@@ -86,6 +88,100 @@ public static (bool isSuccess, IConfig config, CommandLineOptions options) Parse
             return result;
         }
 
+        private static IEnumerable<string> ExpandResponseFile(string[] args)
+        {
+            foreach (var arg in args)
+            {
+                if (arg.StartsWith("@"))
+                {
+                    var fileName = arg.Substring(1);
+                    if (File.Exists(fileName))

```

Comment:
What if the given file doesn't exist? I believe we should warn the user about that rather than just silently ignore such an argument.